chore: v1.0.0-beta.121 (#1000)
Redocly/redocly-cli: docs/changelog.md

## 1.0.0-beta.121 (2023-01-25)

### Features

- Added custom output file option to the `join` command.

> - Added an option to include webhooks to [operation-4xx-response](./rules/operations-4xx-response.md) rule.

Reviewer: ```suggestion
- Added an option to include webhooks to [operation-4xx-response](./rules/operations-4xx-response.md) rule.
- Added a new built-in decorator [info-override](./decorators/info-override.md).
```

> ### Features
> 
> - Added custom output file option to join command
> - Added an option to include webhooks to operation-4xx-response rule
> 
> ### Fixes
> 
> - Ignored case when inferring file extension from codeSample lang property
> 

Reviewer 2: ```suggestion
### Features

- Added custom output file option to the `join` command.
- Added an option to include webhooks to [operation-4xx-response](./rules/operations-4xx-response.md) rule.

### Fixes

- Ignored case when inferring file extension from code sample `lang` property.

```

Reviewer 2: Write in sentences with a full stop.

## 1.0.0-beta.121 (2023-01-25)

### Features

- Added custom output file option to the `join` command.
- Added an option to include webhooks to [operation-4xx-response](./rules/operations-4xx-response.md) rule.

> - Added a new built-in decorator [info-override](./decorators/info-override.md).

Reviewer 2: ```suggestion
- Added a new built-in decorator [info-override](./decorators/info-override.md).
- Added support for `/` as a separator which puts paths into subdirectories for each path segment with the [split command](./commands/split.md).
```
